DailyReportWeb/Controllers/Api/ConfirmDraftController.cs

```diff
@@ -58,7 +61,7 @@ private string GetDraftConfirmationDetails(ExecutionContext context, AdvancedRep
                 var basicSettingsId = db.BasicSettings.Single(bs => bs.UniqueProjectKey == context.Id).Id;
                 var reportSentDate = db.ReportExecutionSummaries.Single(res => res.BasicSettingsId == basicSettingsId).LastFinalReportSentDate;
 
-                if (reportSentDate != null && reportSentDate.Value.Date == DateTime.Today)
+                if (reportSentDate != null && reportSentDate.Value.ToOriginalTimeZone(offsetFromUtc).Date == DateTime.Now.ToOriginalTimeZone(offsetFromUtc).Date)
```

Comment:
Pai eu am folosit doar == . Vad ca in metodele noi, scrise de Vlad, a facut verificarile cu DateTime.Compare. O sa modific sa fie aceeasi metoda peste tot.